inc/class-wp-frontend-profile.php

```diff
@@ -115,7 +115,7 @@ public function instantiate()
          */
         public function load_textdomain()
         {
-            load_plugin_textdomain('wp-front-end-profile', false, WPFEP_PATH . '/languages/');
+            load_plugin_textdomain('wpfep', false, 'wp-frontend-profile/languages');
```

Comment:
Please revert WPFEP_PATH as it refers to plugin path.

Reply:
Ref: https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#loading-text-domain